Training25/Program.cs

```diff
@@ -3,11 +3,28 @@
 // Copyright (c) Metamation India.
 // ------------------------------------------------------------------
 // Program.cs
-// Program on main branch.
+// T11.1: Program that gets number n from command line and prints nth Armstrong number.
 // ------------------------------------------------------------------------------------------------
+using static System.Console;
 namespace Training25;

```

Comment:
```suggestion
using static System.Console;

namespace Training25;
```
Leave a space between namespace and usings